build.gradle

```diff
@@ -62,10 +62,8 @@ subprojects {
             targetExclude '**/build/**', ".gradle/**"
             importOrder(
```

Comment:
Fighting tools is super frustrating - trying to understand your workflow, are you manually placing imports, using auto-add or using the IntelliJ import organizer?  If using the import organizer, there are tweaks [1] that can be pulled into it consistent with the existing rules, how does that impact your preference?

- [1] https://github.com/opensearch-project/opensearch-migrations/pull/846

Reply:
If you take a look at the comments in the linked PR, you'll see a number of my concerns, that remain unaddressed.  I'd like to figure something out to improve this.  One issue is in getting IntelliJ to pickup configs by default, which that [PR didn't address](https://github.com/opensearch-project/opensearch-migrations/pull/846#discussion_r1693248153).  Another way to skin that cat would be to use the default IntelliJ list, which is probably more universally adopted than the opensearch one or the one that the spotless rules use now & use that.

The only issue that I have at the moment is that I can't figure out how to get static imports to work well with spotless (they work find w/ IntelliJ).